### ISSUE TITLE
auto-reload of web page is switched off

### ISSUE THREAD

**Issue by a-knust** [jomjol/AI-on-the-edge-device#2352]
### The Problem

After updating to version 15.2.0 the web pages are no longer automatically reloaded.

### Version

v15.2.0

### Logfile

```shell
nothing available in the logfile
```

**Comment by caco3** [comment]
are you sure this is not an issue with your webbrowser? I am sure this gets handled in the web UI and we did not change anything there!

the web UI us supposed to reload every 5 minutes IIRC.

can you open the dev tools of your browser and show the network log of 15 minutes or so?

Note that this might be repkaced anyway in the future: https://github.com/jomjol/AI-on-the-edge-device/pull/2262

**Comment by a-knust** [comment]
I've updated yesterday from V15.1.1 to 15.2.0. I did some reboots to activate some changes (e.g. modification of ROIs). The automatic refresh was gone directly from the beginning.

Very "interesting" behaviour:
I did a fresh reboot and after completion of reboot I activated the network recording. I can see refreshing the page every 300 seconds (and the content is changed, counter is raised).
<img width="1985" alt="image" src="https://user-images.githubusercontent.com/131944028/234776754-8d922d8e-3dc1-4fef-8b95-4eaf14564fff.png">
After closing the development tools, refreshing continues (counter is raising).
I did again a fresh reboot and after completion of reboot automatic refreshing started again. Maybe the start of the development tools did something like cleaning up the cache in the background.

Behaviour is no longer reproducible, I close the bug ;-)
